### ISSUE TITLE
Configuration issue on react native cli. Not working

### ISSUE THREAD

**Issue by fury-r** [nativewind/nativewind#1045]
**Describe the bug**
NativeWind is not applying styles in a React Native project. When using Tailwind CSS classes like font-bold and text-black, no styles are reflected on the Text component. Even flexbox-related styles such as flex and flexDirection aren't being applied. It seems that NativeWind isn't working as expected, despite following the setup as per the official documentation.

**Reproduction**
Please see our contribution guide: https://github.com/nativewind/nativewind/blob/main/contributing.md#opening-an-issue

Please provide a reproduction of the issue using one of the following templates:

use react-native-cli
package installed
    "nativewind": "^4.1.6",
    "tailwindcss": "^3.4.10", //devDependency
    "react-native-reanimated": "^3.15.1",
    "react-native": "0.75.2",

metro.config.js
```
const {getDefaultConfig, mergeConfig} = require('@react-native/metro-config');
const {withNativeWind} = require('nativewind/metro');

/**
 * Metro configuration
 * https://reactnative.dev/docs/metro
 *
 * @type {import('metro-config').MetroConfig}
 */
const config = mergeConfig(getDefaultConfig(__dirname), {});

module.exports = withNativeWind(config, {input: './global.css'});
```
tailwind.config.js
```
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: [
    './App.{js,jsx,ts,tsx}',
  './src/**/*.{js,jsx,ts,tsx}',  
  ],
  presets: [require('nativewind/preset')],
  theme: {
    extend: {},
  },
  plugins: [],
};

```

babel.config.js
```
module.exports = {
  presets: [
    ['module:@react-native/babel-preset', {jsxImportSource: 'nativewind'}],
    'nativewind/babel',
  ],
  plugins: [
    'babel-plugin-transform-typescript-metadata',
    ['@babel/plugin-proposal-decorators', {legacy: true}],
    'react-native-reanimated/plugin',
  ],
};

```

global.css
```
@tailwind base;
@tailwind components;
@tailwind utilities;
```

nativewind-env.d.ts
```
/// <reference types="nativewind/types" />

```

Test.tsx
```
import React from 'react';
import {Text, View} from 'react-native';

export const AudioRow = () => {
  return (
    <View
      style={{
        display: 'flex',
        flexDirection: 'row',
        alignItems: 'center',
        borderStyle: 'solid',
        borderWidth: 0.5,
        borderRadius: 10,
        marginVertical: 5,
        ...styles.shadowProp,
      }}>

        <Text className="font-bold text-black">Hello world</Text>

    </View>
  );
};


```

**Expected behavior**
font should be bold and color should be black

**Additional context**
It is not working. Not just this even the flex is not working. Nothing from nativeWind is working for me.Can you please help me in telling which configuration is wrong i followed the document properly.[link]( https://www.nativewind.dev/v4/overview)
I even tried nativeWindv2 but that is giving some babel issues

**Comment by thonyromes** [comment]
I am also experiencing this after trying to upgrade gradle to 8.7 (and doing other fixes to get this working like enabling autolinking).

I am on
react-native: 0.74.5
nativewind: ^4.0.1
tailwindcss: @3.4.10

Prior to this when I build, I see tailwind(android) building...
But now nothing happens and no styles get applied both in debug and release builds

I have also tried upgrading react-native to 0.75.2. Everything else works perfectly but nativewind styles.

Currently trying to force gradle to 8.6 and see if it starts working again

**Comment by thonyromes** [comment]
> I am also experiencing this after trying to upgrade gradle to 8.7 and 8.8 (and doing other fixes to get this working like enabling autolinking).
> 
> I am on react-native: 0.74.5 nativewind: ^4.0.1 tailwindcss: @3.4.10
> 
> Prior to this when I build, I see tailwind(android) building... But now nothing happens and no styles get applied both in debug and release builds
> 
> I have also tried upgrading react-native to 0.75.2. Everything else works perfectly but nativewind styles.
> 
> Currently trying to force gradle to 8.6 and see if it starts working again

Unfortunately I couldn't downgrade gradle to 8.6 and gradle WASN'T the problem.
For now, I had to temporarily fix nativewind to v4.0.36 and tailwind to v3.4.6 to get back to my last working state. Everything is updated to current latest version and react-native at v0.75.2. Cheers👍

**Comment by fury-r** [comment]
> > I am also experiencing this after trying to upgrade gradle to 8.7 and 8.8 (and doing other fixes to get this working like enabling autolinking).
> > I am on react-native: 0.74.5 nativewind: ^4.0.1 tailwindcss: @3.4.10
> > Prior to this when I build, I see tailwind(android) building... But now nothing happens and no styles get applied both in debug and release builds
> > I have also tried upgrading react-native to 0.75.2. Everything else works perfectly but nativewind styles.
> > Currently trying to force gradle to 8.6 and see if it starts working again
> 
> Unfortunately I couldn't downgrade gradle to 8.6 and gradle WASN'T the problem. For now, I had to temporarily fix nativewind to v4.0.36 and tailwind to v3.4.6 to get back to my last working state. Everything is updated to current latest version, react-native at v0.75.2 and gradle at 8.8. Cheers👍

I see tailwind is getting built but i can even see the it in the output.css but it is not getting applied.

**Comment by thonyromes** [comment]
> > > I am also experiencing this after trying to upgrade gradle to 8.7 and 8.8 (and doing other fixes to get this working like enabling autolinking).
> > > I am on react-native: 0.74.5 nativewind: ^4.0.1 tailwindcss: @3.4.10
> > > Prior to this when I build, I see tailwind(android) building... But now nothing happens and no styles get applied both in debug and release builds
> > > I have also tried upgrading react-native to 0.75.2. Everything else works perfectly but nativewind styles.
> > > Currently trying to force gradle to 8.6 and see if it starts working again
> > 
> > 
> > Unfortunately I couldn't downgrade gradle to 8.6 and gradle WASN'T the problem. For now, I had to temporarily fix nativewind to v4.0.36 and tailwind to v3.4.6 to get back to my last working state. Everything is updated to current latest version, react-native at v0.75.2 and gradle at 8.8. Cheers👍
> 
> I tried this configuration. I see tailwind is getting built also i can even see the it in the output.css but it is not getting applied.

You have to completely clean all cache, gradle, android builds, node_modules, lock files, etc. I experienced this styles no getting applied on nativewind: 4.1.6

Also confirm your tailwind.config.js, metro.confgi.js, babel.config.js are all configured according to the docs

**Comment by fury-r** [comment]
> > > > I am also experiencing this after trying to upgrade gradle to 8.7 and 8.8 (and doing other fixes to get this working like enabling autolinking).
> > > > I am on react-native: 0.74.5 nativewind: ^4.0.1 tailwindcss: @3.4.10
> > > > Prior to this when I build, I see tailwind(android) building... But now nothing happens and no styles get applied both in debug and release builds
> > > > I have also tried upgrading react-native to 0.75.2. Everything else works perfectly but nativewind styles.
> > > > Currently trying to force gradle to 8.6 and see if it starts working again
> > > 
> > > 
> > > Unfortunately I couldn't downgrade gradle to 8.6 and gradle WASN'T the problem. For now, I had to temporarily fix nativewind to v4.0.36 and tailwind to v3.4.6 to get back to my last working state. Everything is updated to current latest version, react-native at v0.75.2 and gradle at 8.8. Cheers👍
> > 
> > 
> > I tried this configuration. I see tailwind is getting built also i can even see the it in the output.css but it is not getting applied.
> 
> You have to completely clean all cache, gradle, android builds, node_modules, lock files, etc. I experienced this styles no getting applied on nativewind: 4.1.6
> 
> Also confirm your tailwind.config.js, metro.config.js, babel.config.js are all configured according to nativewind v4 docs

That worked.Thanks
